src/components/SharedComponents/Accordion.style.tsx

```diff
@@ -23,13 +25,21 @@ export const StyledAccordionSummary = styled(AccordionSummary)`
 
 export const MarkdownBody = styled(MarkdownContent)`
```

Comment:
suggestion: lets rename this something like `AccordionBodyMarkdown` so it doesnt get confused with the more global `MarkdownBody` component in `components/Markdown/Markdown.style.ts` (and be careful to swap out the right ones after the rename)